### ISSUE TITLE
Error installing in Powershell

### ISSUE THREAD

**Issue by jbrines** [LSPosed/MagiskOnWSA#395]
Hi All,

I get the following error when installing. Tried restarting and still the same error, any ideas on how to resolve?

![image](https://user-images.githubusercontent.com/6060024/171840859-1801dd2f-6d60-42cf-ba01-a7486b820169.png)

**Comment by alex-sandro92** [comment]
Are you running Windows as an admin user?

**Comment by m12adel** [comment]
I have exactly same issue , Power shell is running as admin with also -ExecutionPolicy Bypass attribute

**Comment by alex-sandro92** [comment]
Yes, but does the user account on which you are using windows have admin privileges? Here is the same issue you are facing #233

**Comment by xiaozhiqi1005172** [comment]
Not cd,u must add-package -register first

**Comment by xiaozhiqi1005172** [comment]
> add-package -register AppxManifest.xml
just wait
and 

and then use powershell to run install.ps1

**Comment by xiaozhiqi1005172** [comment]
[run.zip](https://github.com/LSPosed/MagiskOnWSA/files/8839193/run.zip)
unzip it to the apppackage,such as D:\WSA-with-magisk-NoGApps_2204.40000.19.0_x64_Release-Nightly.And right click the run.ps1,run it in the powershell

**Comment by jbrines** [comment]
> Are you running Windows as an admin user?

My normal user is not an admin user. I have tried running powershell as admin.

**Comment by alex-sandro92** [comment]
According to #233 you need to:

1. open control panel
2. change your standard account to an administrator
3. run the install script
4. change your account back to a standard user

**Comment by dalanfsu** [comment]
Powershell starts and closes when running install.ps1
Windows Dev Build 25131.1000
Admin user

**Comment by mfisher224** [comment]
Same issue.

**Comment by bolabola6789** [comment]
> Powershell starts and closes when running install.ps1 Windows Dev Build 25131.1000 Admin user

Same issue, build 22621.
I'm also running as admin, and my system is fully supported by W11 and WSA.
Tried executionpolicy bypass and the powershell window just crashes after around 30 seconds

**Comment by yujincheng08** [comment]
Try to manually `Add-AppxPackage`
